extension-style/api/extension-style.api

```diff
@@ -3667,7 +3667,8 @@ public final class com/mapbox/maps/extension/style/sources/generated/Encoding :
 public final class com/mapbox/maps/extension/style/sources/generated/GeoJsonSource : com/mapbox/maps/extension/style/sources/Source {
 	public static final field Companion Lcom/mapbox/maps/extension/style/sources/generated/GeoJsonSource$Companion;
 	public fun <init> (Lcom/mapbox/maps/extension/style/sources/generated/GeoJsonSource$Builder;)V
-	public synthetic fun <init> (Lcom/mapbox/maps/extension/style/sources/generated/GeoJsonSource$Builder;Lcom/mapbox/geojson/GeoJson;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Lcom/mapbox/maps/extension/style/sources/generated/GeoJsonSource$Builder;Lcom/mapbox/geojson/GeoJson;Ljava/lang/String;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
I'm not sure why we have this line updated as I added parameter to private constructor

Reply:
Just discussed offline - turns out it's generated by Kotlin to allow to the private constructor to be called from Builder, which otherwise wouldn't be possible.
Similar question discussed : https://stackoverflow.com/questions/54613891/why-private-constructor-of-sealed-class-can-be-called-in-sub-class